src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+export const Tabs = React.memo(
+  ({ domID, tabs, selectedTab, onTabSelected }) => (
+    <section id={domID}>
+      <nav className="nav nav-tabs">
+        {tabs.map(tab => (
+          <a
+            key={tab.id}
+            href={`#${domID}`}
+            className={classNames({
+              'nav-item': true,
+              'nav-link': true,
+              active: selectedTab.id === tab.id,
```

Comment:
```suggestion
            className={classNames('nav-item nav-link', {
              active: selectedTab.id === tab.id,
```
Классы которые всегда должны быть можно записывать так